src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public static final int DATE_PART = 0;
+    public static final int NAME_PART = 1;
+    public static final int WORKED_HOURS_PART = 2;
+    public static final int RATE_PART = 3;
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
remove all unnecessary blank lines in method

Reply:
Changed